### ISSUE TITLE
Selection not cleared in editor

### ISSUE THREAD

**Issue by faerot** [elfmz/far2l#2478]
Sometimes when you click between the edges of the line, when mouse press happens on one line and mouse release on another line, you get this:

![image](https://github.com/user-attachments/assets/87938035-be20-49ff-9076-a19015efeb3d)

One line is selected, another is just visual artifact, i.e. if pressed Ctrl+D to delete selected block, you get:

![image](https://github.com/user-attachments/assets/df5f5f10-057b-4b17-8477-c3f8e0d56d38)

The "selection" clears only after that line is redrawn.
Not that I use mouse a lot in editor, it mostly happens when tring to re-focus back to far2l window, which is annoying.

**Comment by faerot** [comment]
Correction: it is not just visual artifact, this is actual selection part, because when typing over remaining selected block it wipes that block.

**Comment by exkrexpexfex** [comment]
@faerot it seems to be an undocumented feature triggered when shift is pressed, see https://github.com/elfmz/far2l/blob/b863878f4eebeac2c29b9fd5a7da9671720dd8ff/far2l/src/editor.cpp#L2725

**Comment by elfmz** [comment]
yep, its a quick adhoc copy into clipboard without affecting cursor position, also works in viewer
